Automattic/jetpack
# Janitorial: add Prettier to help prettify JS code in Jetpack.

#### Changes proposed in this Pull Request:

This PR adds support for Prettier to help prettify JS code in Jetpack.

The original configuration is copied from Calypso.

Down the road, I think we should, like in Calypso, add Prettier to our pre-commit hook to clean things up a bit.

#### Testing instructions:

* Configure Prettier in your editor.
* Run `yarn` 
* See that your editor can auto-fix your code 

#### Proposed changelog entry for your changes:
* none
